org-formation/090-systems-manager/QuickSetup-PatchPolicy.yaml

```diff
@@ -0,0 +1,92 @@
+            Patch Baseline for Amazon Linux 2022 Provided by AWS.","disabled":false},"AMAZON_LINUX_2023":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-05c9c9bf778d4c4d0","label":"AWS-AmazonLinux2023DefaultPatchBaseline","description":"Default
+            Patch Baseline for Amazon Linux 2023 Provided by AWS.","disabled":false},"CENTOS":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-03e3f588eec25344c","label":"AWS-CentOSDefaultPatchBaseline","description":"Default
+            Patch Baseline for CentOS Provided by AWS.","disabled":false},"DEBIAN":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-09a5f8eb62bde80b1","label":"AWS-DebianDefaultPatchBaseline","description":"Default
+            Patch Baseline for Debian Provided by AWS.","disabled":false},"MACOS":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-0ee4f94581368c0d4","label":"AWS-MacOSDefaultPatchBaseline","description":"Default
+            Patch Baseline for MacOS Provided by AWS.","disabled":false},"ORACLE_LINUX":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-06bff38e95fe85c02","label":"AWS-OracleLinuxDefaultPatchBaseline","description":"Default
+            Patch Baseline for Oracle Linux Server Provided by AWS.","disabled":false},"RASPBIAN":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-0ec16280999c5c75e","label":"AWS-RaspbianDefaultPatchBaseline","description":"Default
+            Patch Baseline for Raspbian Provided by AWS.","disabled":false},"REDHAT_ENTERPRISE_LINUX":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-0cbb3a633de00f07c","label":"AWS-RedHatDefaultPatchBaseline","description":"Default
+            Patch Baseline for Redhat Enterprise Linux Provided by AWS.","disabled":false},"ROCKY_LINUX":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-03ec98bc512aa3ac0","label":"AWS-RockyLinuxDefaultPatchBaseline","description":"Default
+            Patch Baseline for Rocky Linux Provided by AWS.","disabled":false},"SUSE":{"value":"arn:aws:ssm:us-east-1:075727635805:patchbaseline/pb-07d8884178197b66b","label":"AWS-SuseDefaultPatchBaseline","description":"Default
+            Patch Baseline for Suse Provided by AWS.","disabled":false},"UBUNTU":{"value":"pb-06e3563bd35503f2b","label":"custom-UbuntuServer-Blog-Baseline","description":"Default
+            Patch Baseline for Ubuntu Provided by AWS.","disabled":false},"WINDOWS":{"value":"pb-016889927b2bb8542","label":"custom-WindowsServer-Blog-Baseline","disabled":false}}'
+          TargetType: '*'
+          TargetOrganizationalRoot: !Ref TargetOrgRoot
+          TargetRegions: !Ref TargetRegions
+Outputs:
+  PatchPolicyArn:
+    Description: 'Patch policy ARN'
+    Value: !Ref PatchPolicyAll
```

Comment:
PatchPolicyAll is not a resource so i don't think you can make a reference to it

Reply:
whoops